### DISCUSSION TITLE
Exporter 400s when colors have Uppercase letters

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#194] [Bug Reports]
This request fails: `https://exporter.battlesnake.com/avatars/head:default/tail:default/color:%23FF4444/120x20.svg`
with a 400 `Bad Request`

However! This one works great! `https://exporter.battlesnake.com/avatars/head:default/tail:default/color:%23ff4444/120x20.svg`

The difference is that the first one uses `FF4444` as the hex color, and the second one uses lower case `f`s: `ff4444`

I noticed this when looking at some games on my Challenges Snakes. I am seeing an Image Alt text instead of the Snake Picture
![image](https://user-images.githubusercontent.com/964771/162512870-65084926-f358-44b6-9faf-48d1822bec45.png)

Thought the exporter might be down, but realized other games were working fine. Noticed the capitalization and tried removing it and the exporter liked that better

**Comment by robbles** [comment]
Hey @coreyja - thanks for finding and reporting this!

This should be fixed now - you can pass uppercase colors to the exporter (and returning them from your snake shouldn't break anything anymore!)

![snake url](https://exporter.battlesnake.com/avatars/head:default/tail:default/color:%23FF4444/120x20.svg)
